### PR TITLE
Fix #2845 unsafe eval on GitHub

### LINKED ISSUE #2845: Chrome extension doesn't work on Github
image

### PR DESCRIPTION
**Pull request by seanh**

### CODE REVIEW

File: h/static/scripts/vendor/frame-rpc.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+var isarray = require('isarray');
```
Comment (robertknight) [inline h/static/scripts/vendor/frame-rpc.js]: The upstream license needs to be reproduced at the top of this file - see https://github.com/substack/frame-rpc/blob/master/LICENSE

Comment (seanh) [inline h/static/scripts/vendor/frame-rpc.js]: Done

**Comment by nickstenning** [comment]
Looks good, but for two issues:

1. If it's been modified by us, it shouldn't be in `vendor/`. `vendor/` implies unmodified code that has been "vendored" into that directory. If we've modified it put it directly in our source tree, possibly with a comment at the top of the file to explain the purpose of the code.

2. You need to replace the use of `isarray` with `angular.isArray`.

**Comment by seanh** [comment]
Both done, tests should pass now

**Comment by robertknight** [comment]
`angular.isArray()` is fine for the moment but we can now use the standard library's `Array.isArray` function.

**Comment by judell** [comment]
Happy dance!

**Pull request merged**